index.js

```diff
@@ -57,7 +57,11 @@ function defaultConsumer (field, file, filename, encoding, mimetype, body) {
   file.on('data', data => { fileData.push(data) })
```

Comment:
we could add the same check here in order to avoid memory usage:

```suggestion
  file.on('data', data => { if (!lastFile.limit) { fileData.push(data) } })
```